middleware/service/unittest/source/manager/test_manager.cpp

```diff
@@ -1226,5 +1226,43 @@ namespace casual
          }
       }
 
+      TEST( service_manager, runtime_update_routes)
```

Comment:
In what regard?

Reply:
I wondered if maybe it should test that the whole state has been correctly updated, rather than just the routes, but I'm not sure.

Reply:
> I wondered if maybe it should test that the whole state has been correctly updated, rather than just the routes, but I'm not sure.

I think it's better to be finer then coarser.